compiler/src/iree/compiler/Codegen/Passes.h

```diff
@@ -715,9 +715,12 @@ createSPIRVBreakDownLargeVectorPass();
 std::unique_ptr<OperationPass<func::FuncOp>>
 createSPIRVCreateFastSlowPathPass();
 
-/// Emulates 64-bit integer ops with 32-bit integer ops.
+/// Emulate 64-bit integer ops with 32-bit integer ops.
```

Comment:
Do we prefer `Enable` over `Enables`? I thought that the google style encourages the previous style: https://developers.google.com/style/reference-verbs